### PR TITLE
Migrate image commands to use refactored powershell modules for host variant

### PR DESCRIPTION
**Pull request by abhitoshj**
Migrated all image commands to use the migrated image commands for the host variant of k2s.

### CODE REVIEW

File: lib/scripts/k2s/image/Build-Image.ps1 [hunk 1]
```diff
@@ -451,9 +481,13 @@ if ($Push) {
 # Cleanup inside VM
 if (!$Keep -and !$Windows) {
     Write-Log 'Cleaning up temporary disk space in control plane VM' -Console
-    Invoke-CmdOnControlPlaneViaSSHKey 'find ~/tmp/docker-build -exec chmod a+w {} \; ; rm -rf ~/tmp/docker-build'
+    #Invoke-CmdOnControlPlaneViaSSHKey 'find ~/tmp/docker-build -exec chmod a+w {} \; ; rm -rf ~/tmp/docker-build'
```
Comment (jgriegershs) [inline lib/scripts/k2s/image/Build-Image.ps1:484]: Why is this commented out?

File: lib/scripts/k2s/image/Clean-Images.ps1 [hunk 2]
```diff
@@ -14,25 +14,43 @@ Remove all container images in K2s
...
+    [parameter(Mandatory = $false, HelpMessage = 'If set to true, will encode and send result as structured data to the CLI.')]
+    [switch] $EncodeStructuredOutput,
+    [parameter(Mandatory = $false, HelpMessage = 'Message type of the encoded structure; applies only if EncodeStructuredOutput was set to $true')]
+    [string] $MessageType
 )
 
 $clusterModule = "$PSScriptRoot/../../../modules/k2s/k2s.cluster.module/k2s.cluster.module.psm1"
 $infraModule = "$PSScriptRoot/../../../modules/k2s/k2s.infra.module/k2s.infra.module.psm1"
-Import-Module $clusterModule, $infraModule
+$statusModule = "$PSScriptRoot/../../../modules/k2s/k2s.cluster.module/k2s.cluster.module.psm1"
```
Comment (jgriegershs) [inline lib/scripts/k2s/image/Clean-Images.ps1:26]: Cluster module is already imported by line 24.

**Comment by achimkist** [comment]
https://github.com/Siemens-Healthineers/K2s/blob/b63971816b8dd5743f079cb0024843f4dba73161/lib/modules/k2s/k2s.cluster.module/image/image.module.psm1#L410

still globals in image.module.ps1

File: lib/scripts/k2s/image/Build-Image.ps1 (hunk 1 above)
Comment (abhitoshj) [reply lib/scripts/k2s/image/Build-Image.ps1:484]: I had disabled the folder deletion for debugging one issue. This should be uncommented.

**Review by achimkist — APPROVED** [review]

**Pull request merged**